slackclient/_client.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, token):
         self.token = token
         self.server = Server(self.token, False)
 
+    def get_user_agent(self):
```

Comment:
I removed these to make this functionality more consistent with what Don implemented in the Node lib.